admin-events/package-lock.json

```diff
@@ -8,12 +8,7 @@
       "name": "forta-aave-admin-events",
```

Comment:
I received a warning for 3 vulnerabilities when I used `npm i`.

When I removed this `package-lock.json` file altogether and allowed `npm` to create a new one, the warning went away.  Perhaps we should just start with a fresh `package-lock.json` file?

Reply:
I ran `npm audit fix` and committed the updated `package-lock.json`